internal/util/util.go

```diff
@@ -158,6 +157,41 @@ type KernelVersion struct {
 	Backport     bool   // backports have a fixed version/patchlevel/sublevel
 }
 
+// Parses a kernel release version string into:
+//   version, patch version, sub version and extra version
```

Comment:
@nit:  may be remove first space in above comment.

Reply:
Done.